packages/quick_actions/quick_actions_platform_interface/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
2013 The Flutter Authors

Same for all files in this PR, otherwise whichever of this and https://github.com/flutter/plugins/pull/3737 lands second will break the tree.

Reply:
Fixed this in all the project files